modules/rampart-trust/src/main/java/org/apache/rahas/impl/SAML2TokenIssuer.java

```diff
@@ -948,7 +948,8 @@ protected Assertion createBearerAssersion(SAMLTokenIssuerConfig config,
     }
 
     public String getResponseAction(RahasData data) throws TrustException {
-        return null;
+        return TrustUtil.getActionValue(data.getVersion(),
```

Comment:
Thanks for pointing it out. Fixed with the following PR - https://github.com/wso2/wso2-rampart/pull/58